java/src/main/native/src/TableJni.cpp

```diff
@@ -19,75 +19,119 @@
 
 #include <unordered_set>
```

Comment:
This might not be needed anymore, since the code that used `unordered_set` has been deleted.